gradle.properties

```diff
@@ -14,7 +14,7 @@ protocVersion = 3.17.2
 # plugins
 #
 versionsPluginVersion = 0.39.0
-versionsFilterPluginVersion = 0.1.10
+versionsFilterPluginVersion = 0.1.4
```

Comment:
this seems to be problematic to me, like get stuck with an older version of something and not getting fixed updates

Reply:
yeah, i understand. But it's the only simple way. Otherwise we should find other plugin, or implement ourselves

Reply:
I understand too but bear in mind that there's no many developer working on this project so I really need to keep the maintenance cost as low as possible. In this particular case I'm fine to provide a custom implementation as it should be very simple but in general, I don't have the time to try to keep up with all those details.

If anyone wants to step in, I'm more than happy

Reply:
Yes, i understand it very well and would like to help you to support this project. Ok, i'l try to create custom implementation without this plugin